packages/ui/atomic/create-atomic-component/template/src/components/sample-component/stencil.config.ts

```diff
@@ -13,6 +13,10 @@ export const config: Config = {
     {
       type: 'docs-readme',
     },
+    {
+      type: 'docs-json',
+      file: 'docs/atomic-docs.json',
```

Comment:
Like my comment on the other PR, I think we should make this name less specific to Atomic since it's just docs generated by Stencil in our template.

We could name it something like `stencil-docs.json` instead.